tests/prometheus/prometheus.go

```diff
@@ -0,0 +1,45 @@
+package prometheus
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/kudobuilder/test-tools/pkg/client"
+	"github.com/mesosphere/kudo-cassandra-operator/tests/curl"
+)
+
+type PrometheusSearch struct {
+	Status string               `json:"parameters"`
+	Data   PrometheusSearchData `json:"data"`
+}
+
+type PrometheusSearchData struct {
+	Type   string                   `json:"resultType"`
+	Result []PrometheusSearchResult `json:"result"`
+}
+
+type PrometheusSearchResult struct {
+	Metric map[string]interface{} `json:"metric"`
+	Value  []interface{}          `json:"value"`
+}
+
+func QueryForStats(client client.Client, namespace string, baseURL, query string) (*PrometheusSearch, error) {
```

Comment:
Does the namespace matter here? It's [already part of the DNS](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#namespaces-and-dns), or, in case of a FQDN, namespaces shouldn't matter.

Reply:
It doesn't, really. It was only required for the Curl-Runner, to know in which namespace to launch the temporary pod. I've refactored that so it's not part of the QueryForStats anymore